git/gitgithubclient.py

```diff
@@ -40,7 +39,7 @@ def _commitData(self, commit):
 			"message": commit.commit.message
 		}
 
-	def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.timedelta(hours=24)):
+	def commits(self, branch, since=0, to=0):
```

Comment:
Agree it is redundant to convert one format to another and back. I prefer timestamp as in most case gitclient will be used instead of github/Atlassian REST API. In general, the client will be used for two use cases:

* give me all commits starting from a given date
* give me all commits in a given date interval

For the former, datetime is better as you don't have to convert from timestamp.
For the latter, timestamp is better as you call the client with commit dates in timestamp form.

Sadly, the git module requires datetime, atlassian API returns datetime.
I prefer timestamp is you compare two integers. For datime you compare to strings.

For the sake of consistency let's stick with timestamps for the moment and deal with the date format later.